client/overview/index.js

```diff
@@ -144,11 +145,25 @@ const OverviewPage = () => {
 			{ ! accountRejected && (
 				<ErrorBoundary>
 					<>
-						<AccountBalances
-							numDisputesNeedingResponse={
-								numDisputesNeedingResponse
-							}
-						/>
+						{ 0 < numDisputesNeedingResponse ? (
```

Comment:
This is a strange way to order the statement IMO, but enforced by the "yoda condition" eslint rule.

Reply:
Can always use a local variable to make this less awkward, e.g. 

```js
const someDisputesNeedAttention = 0 < numDisputesNeedingResponse;
```

Reply:
No longer relevant since we're not splitting card in this PR. 8b31b06